test/contract_qearn.cpp

```diff
@@ -146,7 +175,7 @@ class ContractTestingQearn : protected ContractTesting
         return output.returnCode;
     }
 
-    sint32 unlock(const id& user, long long amount, uint16 lockedEpoch)
+    sint32 unlock(const id& user, long long amount, uint32 lockedEpoch)
```

Comment:
Usually, the epoch is store in uint16 in the core, see qpi.h for example.

Reply:
I know that. also I already used uint16 for epoch. but I couldn't request to SC and make the transaction using uint16 parameter. it is returned wrong result.

Reply:
The wrong results may have been due to alignment in the structs differing between client and server side.

Reply:
I have checked many times about this case. at first I tested with uint16 lockedEpoch. at that time the SC can't get the value from frontend. so returned wrong result. so I tried to change only uint16 to uint32 and tested the SC. at this time, returned correct result.